scripts/ts-utils/add-whitelist-script.ts

```diff
@@ -0,0 +1,56 @@
+
+//run:npx ts-node add-whitelist-script.ts
+
+import { initApi } from "./index";
+
+let whitelist = ['5EtChN6SSfL7E13fEqKQn4jZztDtZkSiRqfFTv2BKufL5fzZ'];
```

Comment:
This 'whitelist' should be either an input parameter or to read from a file. I prefer the later option.

Reply:
fixed